pkg/snapshot/layered_map.go

```diff
@@ -85,11 +102,18 @@ func (l *LayeredMap) MaybeAddWhiteout(s string) (bool, error) {
 
 // Add will add the specified file s to the layered map.
 func (l *LayeredMap) Add(s string) error {
```

Comment:
is the name of the file or contents?

Reply:
`s` is the name of the file

Reply:
sorry, is that what you were asking?

Reply:
yes. i was just wondering if we shd be hashing the file contents as well.

Reply:
we call the hasher function on line 106:
```
newV, err := l.hasher(s)
```
and that function will read the file contents, modtime, and some other things as well

Reply:
hmm.. After reading the PR description again, i think the cacheHasher does it.

Reply:
yah, it does!